src/lib.rs

```diff
@@ -830,6 +814,34 @@ impl<WP: ServerGatewayApis + Send + Sync + 'static> CoreSDK<WP> {
     }
 }
 
+impl<WP> CoreSDK<WP>
+where
+    WP: ServerGatewayApis + Send + Sync + 'static,
+{
```

Comment:
This impl doesn't need to be redeclared. Same as the above one.

Reply:
Done